README.md

```diff
@@ -0,0 +1,16 @@
+#### Data files for [MetabolicAtlas.org](https://metabolicatlas.org)
+
+This repository centralizes data hosted in other repositories to facilitate the deployment of Metabolic Atlas. The long-term vision is to automate the deployment process, which would circumvent the need to duplicate these respositories.
```

Comment:
It's not clear how automating the deployment process leads to not needing to duplicate the repos. Perhaps we could add some more details about that, or remove this sentence for now.

Reply:
I've added a few more words there, is the message clearer now?

Reply:
It helps, but does that mean this `data-files` repo would become unnecessary when the automation is in place?

Reply:
Maybe, it's hard to say prior to the implementation of this automation.

Reply:
Ok, I personally would choose to either expand even more about it or remove the mention of automating the deploy. But it's a minor issue and I think it's good to let people know that we are aware of the duplication at least.